src/Stoa.ts

```diff
@@ -38,6 +39,11 @@ class Stoa extends WebService
      */
     public coinMarketService : CoinMarketService;
 
+    /**
+     * Stoa Block height 
+     */
+    public static height: Height;
+
```

Comment:
I think this is for log function, but isn't it better to add it to './modules/common/Logger'?

Reply:
yes this is related to logger but we need height of stoa at which  logs was stored. as it is related to stoa so it is better to have this variable in stoa class.

Reply:
I want to resolve the dependency.
After creating a class that only has the status of the block, it would be nice to manage the shared information with a single tone.

Reply:
Let's go over this again later.

Reply:
i modified